src/Umbraco.Core/Models/DataType.cs

```diff
@@ -192,5 +192,13 @@ public void SetLazyConfiguration(string? configurationJson)
                 });
             }
         }
+
+        public IDataType DeepCloneWithResetIdentities()
```

Comment:
Since it is the same implementation as interface default, this is not really necessary 😁

Reply:
You're right. Has been removed!